### DISCUSSION TITLE
Feature ideas

### DISCUSSION THREAD

**Discussion by dj1ch** [dj1ch/minigotchi#73] [Ideas]
Hey everyone, it's me @dj1ch. I would like to ask the community on the behalf of myself and others, but in this case I want to hear some ideas for some features with the Minigotchi. So far the new release is almost done and I hope to get it done by this weekend. I might not be able to add them by the release coming, but they most likely will be implemented by the following one. Again, thank you for all the time and support for this project

**Comment by macgyverfuerarme** [comment]
I'm still hoping for a display. Either via the Flipper Zero or with your own e-ink display. Keep up the good work. Thank you for your effort

**Reply by dj1ch** [reply]
I'll look into soldering pins onto the d1. Thanks for the support

**Reply by dj1ch** [reply]
another question, do you have pins soldered to your ESP8266? when it comes to any arduino board with gpio pins like the flipper i think you can get the output from the board... but i think i can probably adjust the code for flipper support. there might be another way to read info through the gpio.

**Comment by LavedenC1** [comment]
Is there the deep reinforcement AI and the WiFi cracking?

**Reply by dj1ch** [reply]
No, there won't be since the ESP8266 and ESP32 don't have the power to do that. TF can run on microcontrollers but I don't plan on any sort of implementation. WiFi cracking is impossible considering we can't get PCAPs with a Minigotchi, only a Pwnagotchi can. In terms of power microcontrollers are very weak.
